### ISSUE TITLE
How to get the loss of validation set

### ISSUE THREAD

**Issue by ywj509** [open-mmlab/mmdetection#6387]
{"mode": "train", "epoch": 1, "iter": 100, "lr": 0.0005, "memory": 2421, "data_time": 0.00314, "loss_rpn_cls": 0.45784, "loss_rpn_bbox": 0.03336, "s0.loss_cls": 0.0667, "s0.acc": 98.36328, "s0.loss_bbox": 0.01117, "s1.loss_cls": 0.04274, "s1.acc": 98.375, "s1.loss_bbox": 0.00517, "s2.loss_cls": 0.02605, "s2.acc": 98.38086, "s2.loss_bbox": 0.00267, "loss": 0.6457, "time": 0.31286}
{"mode": "val", "epoch": 1, "iter": 101, "lr": 0.0005, "AP50": 0.045, "mAP": 0.04545}

How to get the loss of verification set in log file？
